test/from_json.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* global describe, it, before */
+
+var assert = require('assert');
+var CAWParser = require("../build/Release/aw_node_addon").CAWParser;
```

Comment:
As a followup you might want to standardize to Standard JS since we use that in our other repos, but not that important.